tests/test_middleware_interface.py

```diff
@@ -71,6 +71,9 @@
                                             "${PROMPT_PROCESSING_DIR}/tests/data/SingleFrame.yaml",
                                             ],
                               }])
+pipelines_minimal = PipelinesConfig([{"survey": "SURVEY",
+                                      "pipelines": ["${PROMPT_PROCESSING_DIR}/tests/data/ISR.yaml",],
```

Comment:
Really minor, but I think this is missing whitespace before the closing ]

Reply:
Flake8 doesn't complain either way. 🤔